### PR TITLE
To fix cpu_only part

### PR DESCRIPTION
**Pull request by FyodorK**
- Update cpu_only part dockerfile; 
- Fix issue with setup.py install with --cpy-only opt; 
- Fix README.md against Ubuntu 18 requirements

### CODE REVIEW

**Comment by FyodorK** [comment]
Additional logs
- execution python setup.py install --cpu-only before changes: 
http://134.191.240.124/job/nncf/job/precommit/job/manual_precommit/105/console
- after changes:
http://134.191.240.124/job/nncf/job/precommit/job/manual_precommit/106/console
- building docker from docker/cpu/Dockerfile: 
[build_cpu_ub18.log](https://github.com/openvinotoolkit/nncf_pytorch/files/5439376/build_cpu_ub18.log)
- running image:
[run_img_cpu_ub18.log](https://github.com/openvinotoolkit/nncf_pytorch/files/5439378/run_img_cpu_ub18.log)

**Comment by vshampor** [comment]
Remove all "to"s from the commit message.

File: docker/cpu/Dockerfile [hunk 1]
```diff
@@ -39,4 +42,4 @@ ENV no_proxy=$no_proxy
 
 RUN pip install setuptools
 
-ENTRYPOINT cd /home/nncf/nn-compression-framework/ && python setup.py install --cpu-only  && bash
+ENTRYPOINT cd /home/nncf/ && python setup.py develop --cpu-only && bash
```
Comment (vshampor) [inline docker/cpu/Dockerfile:45]: What was wrong with the 'install' setup type?

**Comment by FyodorK** [comment]
Jenkins please retry a build

File: docker/cpu/Dockerfile (hunk 1 above)
Comment (FyodorK) [reply docker/cpu/Dockerfile]: It was nothing wrong actually. I've just needed to test it for sure and forget to change back. This is fixed in next commit.

**Review by vshampor — APPROVED** [review]

**Pull request merged**